src/coreclr/System.Private.CoreLib/CreateRuntimeRootILLinkDescriptorFile.targets

```diff
@@ -1,7 +1,7 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
   <PropertyGroup>
-    <CompileDependsOn Condition="'$(DesignTimeBuild)' != 'true'">$(CompileDependsOn);_CreateILLinkRuntimeRootDescriptorFile</CompileDependsOn>
+    <CoreCompileDependsOn Condition="'$(DesignTimeBuild)' != 'true'">$(CoreCompileDependsOn);_CreateILLinkRuntimeRootDescriptorFile</CoreCompileDependsOn>
```

Comment:
Generally, I thought the guidance was https://github.com/dotnet/sourcelink/issues/392#issuecomment-528391997 to do a BeforeCompile for codegen. Is this coming from any behavior worth commenting or any new documentation?